Source/wi_stuff.c

```diff
@@ -1945,7 +1945,7 @@ void WI_DrawBackground(void)
   else if (gamemode == commercial || wbs->epsd >= 3)
     strcpy(name, "INTERPIC");
   else 
-    sprintf(name, "WIMAP%hhu", (byte) wbs->epsd);
+    snprintf(name, sizeof(name), "WIMAP%d", wbs->epsd);
```

Comment:
How about `WIMAP%.1d`?

Reply:
When and why did this become an issue at all?

Reply:
I don't get it too. Apparently, this is a thing `Werror=format-truncation`.

Reply:
Gosh! Could you revert to the initial printf line and just increase the buffer size to say 32.